app/lib/pre_assembly/batch.rb

```diff
@@ -38,12 +38,6 @@ def run_pre_assembly
       log "\nfinishing run_pre_assembly(#{info_for_log})"
     end
 
-    # used by discovery report
-    def object_filepaths_unique?(dobj)
-      filepaths = dobj.object_files.map(&:path)
-      filepaths.count == filepaths.uniq.count
-    end
-
     # Discovers the digital object containers and the stageable items within them.
```

Comment:
this method was moved to the `ObjectFileValidator` class below, since that is where it really belongs (it is only used when validating things from discovery reports)